Modules/Exercise05/Tests/Unit/Services/OrderServiceTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Modules\Exercise05\Tests\Unit\Services;
+
+use Modules\Exercise05\Services\OrderService;
+use Tests\TestCase;
+
+class OrderServiceTest extends TestCase
+{
+    protected $orderService;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->orderService = new OrderService();
+    }
+
+    /**
+     * @param $detailOrder
+     * @param $expectedValue
+     * @dataProvider provideDetailOrder
+     * */
+    public function test_handle_discount($detailOrder, $expectedValue)
+    {
+        $resultOrder = $this->orderService->handleDiscount($detailOrder);
```

Comment:
Trong provider này nên thêm có 1 case nữa check độ chính xác tới 2 chữ số thập phân vì cái round nó có thêm tham số thứ 2 là `2`
```
$infoBill['price'] = round(($detailOrder['price'] * 80) / 100, 2);
```
Cái này nice to have :smile: 

Ngoài ra nữa thì anh để ý các logic `>=` nhá